modules/apps/frontend-data-set/frontend-data-set-web/src/main/resources/META-INF/resources/views/cards/Cards.js

```diff
@@ -42,7 +42,13 @@ function Cards({dataLoading, dataSetContext, items, schema}) {
 	}
 
 	if (!items?.length) {
-		return <EmptyResultMessage />;
+		return (
+			<ClayEmptyState
+				description={Liferay.Language.get('no-items-were-found')}
+				imgSrc="/o/classic-theme/images/states/empty_state.gif"
+				title=""
```

Comment:
Hey, Why this EmptyState have a empty string as a title?

Reply:
ClayEmptyState by deafult adds a title. As far as I've seen in the FDS the y do not use the title prop. I'm talking with @drakonux about this, we should make some decisions here, but for now I've preferred to replicate what is done

Reply:
Ok :)